beacon_node/beacon_chain/src/blob_verification.rs

```diff
@@ -135,17 +135,20 @@ fn verify_data_availability<T: BeaconChainTypes>(
         .as_ref()
         .ok_or(BlobError::TrustedSetupNotInitialized)?;
 
-    if !kzg_utils::validate_blobs_sidecar(
-        kzg,
-        block_slot,
-        block_root,
-        kzg_commitments,
-        blob_sidecar,
-    )
-    .map_err(BlobError::KzgError)?
-    {
-        return Err(BlobError::InvalidKzgProof);
-    }
+    // TODO: use `kzg_utils::validate_blobs` once the function is updated
+    // I believe this is currently being worked on in another branch.
+    //
+    // if !kzg_utils::validate_blobs_sidecar(
+    //     kzg,
+    //     block_slot,
+    //     block_root,
+    //     kzg_commitments,
+    //     blob_sidecar,
+    // )
+    // .map_err(BlobError::KzgError)?
+    // {
+    //     return Err(BlobError::InvalidKzgProof);
+    // }
```

Comment:
can you mark this with an `unimplemented!`or `todo!` ? (simply easier to find since we have TODO's all over the place in unstable + runtime panic if hit)

Reply:
Updated to use `todo!`.

My original thinking of this one was to not break the existing function, i.e. return `Ok` without validating the blobs, but I guess this entire function needs to be updated to accomodate the blob decoupling changes anyways, so probably not point trying to keep it alive 😅